projects/stargazer/inst/freebsd/etc/stargazer/conf-available.d/store_mysql.conf

```diff
@@ -23,4 +23,10 @@
     # Value: any, supported by database
     # Default: 123456
     Password = 123456
+
+    # Database port
+    # Parameter: optional
+    # Value: any, supported by database
+    # Default: 0
+    Port = 0
```

Comment:
Please comment-out this line, the parameter is optional. Besides, 0 is a wrong value.